src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -2938,11 +2918,13 @@ namespace cryptonote
       uint32_t limit;
       bool active_only;
       requested_fields_t fields;
+      std::string if_block_not_equal;
```

Comment:
Should annotate this field (and the above `active_only`, `limit`) using this field changes the behaviour quite abit and it's not obvious immediately in the docs anymore.

Something like `// Optional: The given block hash is checked against the current service node list and avoids generating output if the hashes are the same (i.e. the list has not changed since the block hash)` 

Also suggestion for name to be `expected_block_hash`

Reply:
Added annotations, and renamed to `poll_block_hash` which better described the use/purpose.

Reply:
Added annotations, and renamed to `poll_block_hash` which better described the use/purpose.